allennlp/commands/elmo.py

```diff
@@ -97,7 +97,7 @@ def add_subparser(self, name: str, parser: argparse._SubParsersAction) -> argpar
         subparser = parser.add_parser(
                 name, description=description, help='Create word vectors using a pretrained ELMo model.')
 
-        subparser.add_argument('input_file', type=argparse.FileType('r'), help='The path to the input file.')
+        subparser.add_argument('input_file', type=argparse.FileType('r', encoding='utf-8'), help='The path to the input file.')
```

Comment:
We include pylint checks in our CI: [this line](http://build.allennlp.org/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=13333&_focus=2529) is saying you need to format this like:

```
subparser.add_argument('input_file', 
					   type=argparse.FileType('r', encoding='utf-8'),
					   help='The path to the input file.')

```

because it is too long. Thanks!!

Reply:
Ah, I see.
Cool!
Thanks!